src/app/routes/utils/constructPageFetchUrl/index.ts

```diff
@@ -214,9 +229,13 @@ const constructPageFetchUrl = ({
         const variantPath = variant ? `/${variant}` : '';
         const host = `http://${process.env.HOSTNAME || 'localhost'}`;
         const port = process.env.PORT ? `:${process.env.PORT}` : '';
-        fetchUrl = Url(
-          `${host}${port}/api/local/${service}/live/${id}${variantPath}`,
-        );
+        let constructedUrl = `${host}${port}/api/local/${service}/live/${id}${variantPath}`;
```

Comment:
Shorter alternative to achieve stripping /${service/live from the id:

const newId = id.replace(/^\/?[^/]+\/live\//, '');
        
        fetchUrl = Url(
          `${host}${port}/api/local/${service}/live/${newId}${variantPath}`,
        );